test/datafit.jl

```diff
@@ -23,7 +23,7 @@ tspan = (0.0, 100.0)
 prob = ODEProblem(sys, u0, tspan, p, jac = true)
 sol = solve(prob)
 
-tsave = [1.0, 2.0, 3.0]
+tsave = [1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0]
```

Comment:
beyond a lyopunov time will not give good results for a chaotic system.

If you want to change the model, I'd suggest lotka-volterra as a better example